monitoring/src/main/java/org/jlab/clas12/monitoring/monitor2p2GeV.java

```diff
@@ -2,40 +2,60 @@
 import java.io.*;
 import java.util.*;
 
-import org.jlab.groot.math.*;
 import org.jlab.groot.data.H1F;
 import org.jlab.groot.data.H2F;
 import org.jlab.groot.math.F1D;
-import org.jlab.groot.fitter.DataFitter;
 import org.jlab.io.base.DataBank;
 import org.jlab.io.base.DataEvent;
 import org.jlab.io.hipo.HipoDataSource;
-import org.jlab.groot.fitter.ParallelSliceFitter;
 import org.jlab.groot.graphics.EmbeddedCanvas;
 import org.jlab.groot.data.GraphErrors;
 import org.jlab.groot.data.TDirectory;
 import org.jlab.clas.physics.Vector3;
 import org.jlab.clas.physics.LorentzVector;
 import org.jlab.groot.base.GStyle;
 import org.jlab.utils.groups.IndexedTable;
-import org.jlab.detector.calib.utils.CalibrationConstants;
 import org.jlab.detector.calib.utils.ConstantsManager;
 
 public class monitor2p2GeV {
+        
+    static final int RGD_ELECTRON_BIT_OFFSETS = 0x4081;
+        
+    /**
+     * @param sector
+     * @return whether one of the "electron" trigger bits for the given
+     * sector was set
+     */
+    boolean testTriggerSector(int sector) {
+        // we've already got run number stuff embedded everywhere, so
+        // why not ...
+        if (runNum > 18300 && runNum < 20000) {
```

Comment:
Added this to https://github.com/JeffersonLab/clas12-timeline/issues/74

We'll need to remember to adjust the upper bound `20000` when we switch to RG-K.

Do we anticipate needing this fix for future run groups?

Reply:
I think for the upcoming RG-K and future runs, we anticipate going back to the way things were up until RG-D, where bit 0 (and 1-7) are the primary trigger or at least usable for standard monitoring utiliies.

I chose 20k because most likely RG-D will cross 19k, and then we'll do the usual bump to the next millenium for the next run group.